api/envoy/extensions/geoip_providers/maxmind/v3/maxmind.proto

```diff
@@ -39,4 +40,11 @@ message MaxMindConfig {
   // Common provider configuration that specifies which geolocation headers will be populated with geolocation data.
   common.v3.CommonGeoipProviderConfig common_provider_config = 4
       [(validate.rules).message = {required: true}];
+
+  // Optional full file path to the config country mappings overrides, e.g. /etc/country-mapping.conf.
+  // File expected to have .conf extension. File should be a json with the following format:
+  // {
+  //   "country_code": "country_code" # comment
+  // }
```

Comment:
```suggestion
  // The file is expected to have a ``.conf`` extension, and should be JSON with the following format:
  //
  // .. code-block::json
  //   {
  //     "country_code": "country_code" # comment
  //   }
  //
```

not 100% if this will resolve the current docs build issues but i think we want to do this anyway